views/includes/check-your-answers/update/update-managing-officer-individual.html

```diff
@@ -4,13 +4,23 @@
 {% set inChangeJourney = not inNoChangeJourney %}
 {% set isNewMo = not (moi.ch_reference | length) %}
 {% set isIndividualManagingOfficer = true %}
+{% set isRegistrationJourney = pageParams.isRegistration %}
 
 {% set moId = moi.id %}
 {% set moFirstName = moi.first_name %}
 {% set moLastName = moi.last_name %}
 {% set moHasFormerNames = moi.has_former_names == 1 %}
 {% set moFormerNames = moi.former_names %}
-{% set moDateOfBirth = dateMacros.formatDate(moi.date_of_birth["day"], moi.date_of_birth["month"], moi.date_of_birth["year"]) %}
+{% set moDateOfBirth %}
+  {% if moi.date_of_birth %}
+    {% if not isRegistrationJourney and moi.date_of_birth["day"] == "01"%}
```

Comment:
the file name seems to suggest this is never for registration but no real harm in checking.

Reply:
Yeah good spot, so there should be no need for the check on registration here

Reply:
👍